openrndr-shape/src/commonMain/kotlin/org/openrndr/shape/internal/BezierCubicSampler2D.kt

```diff
@@ -6,7 +6,7 @@ import kotlin.math.abs
 import kotlin.math.atan2
 
 internal class BezierCubicSampler2D {
-    private val recursionLimit = 8
+    private val recursionLimit = 12
```

Comment:
Limit is raised quite a bit. I guess this won't have any performance impacts outside of testing as I don't think we ever reached the old limit at 8.